backend/controllers/postController.js

```diff
@@ -0,0 +1,92 @@
+import mongoose from "mongoose"
+const ObjectId = mongoose.Types.ObjectId
+import Post from "../models/postModel.js"
+import Comment from "../models/commentModel.js"
+
+async function newPost(req, res, next) {
+    const {
+        content, 
+        takenWith, 
+        createdAt
```

Comment:
This is pre implemented in mongo since we are using the timestamp feild in the schema which by default will create and manage the createdAt and updatedAt
![image](https://github.com/aldab2/family_space/assets/34516271/b32edb42-704c-48fe-a36b-05b6c757dcdd)
![image](https://github.com/aldab2/family_space/assets/34516271/d00f83e5-8837-409e-96fe-07f43e2cc0ec)